core/arch/arm32/sta/sst_km_tests.c

```diff
@@ -0,0 +1,353 @@
+/*
```

Comment:
I can put host side test program in optee_test, but for the secure world TA, if it becomes a user space TA, it will have no chance to access the key manager APIs. So for the secure world TA, it should be a static TA instead.  What do you think, @jbech-linaro ?